src/plugins/data/public/search/expressions/esaggs.ts

```diff
@@ -137,46 +136,35 @@ const handleCourierRequest = async ({
   requestSearchSource.setField('filter', filters);
   requestSearchSource.setField('query', query);
 
-  const reqBody = await requestSearchSource.getSearchRequestBody();
-
-  const queryHash = calculateObjectHash(reqBody);
-  // We only need to reexecute the query, if forceFetch was true or the hash of the request body has changed
-  // since the last request
-  const shouldQuery = forceFetch || (searchSource as any).lastQuery !== queryHash;
```

Comment:
Should we be removing the `forceFetch` request handler param as well, since it doesn't do anything at this point?